.gitignore

```diff
@@ -84,3 +84,5 @@ corehq/apps/hqwebapp/static/hqwebapp/js/lib/modernizr.js
 # direnv
 .envrc
 .direnv
+
+scripts/staging.yml
```

Comment:
nit: I might leave the file in here, but replace it with a big "HEY, YOU SHOULD BE LOOKING AT HTTPS://..." to ease the transition for those who aren't paying close attention.  I think adding it to gitignore means that it'll still be present in people's envs until they delete it, which could get confusing.

Reply:
I think it's a good idea to leave a stub _staging.yml_ with directions to the real _commcare-hq-staging.yml_ file, but that complicates things because `scripts/rebuildstaging` overwrites it. Would it make sense to have `scripts/rebuildstaging` download it to a different location, or maybe update `git-build-branch` to accept a URL?

Reply:
Could just download it to a tmp file that gets deleted after `git-build-branch` completes. Agreed that the staging.yml should be left in place with a notice explaining where the new file lives.

Reply:
Made some changes in d65222501e1c207a71920b9f92c0f456f1497a01. To Ethan's point above about removing the reference to staging.yml, I removed that entire block since now there is no concern around the staging.yml file being out of sync since we do a fresh download for every run of `rebuildstaging`.

Additionally, I restored the staging.yml file with a notice of where to find the new file, and saved the downloaded file to a temp file that gets cleaned up after `git-build-branch` completes.

Reply:
Also meant to say that if we did update `git-build-branch` to accept a url, that tool would still be responsible for creating and deleting the temp file right? It does simplify it from the caller's perspective, but also content with this solution.

Reply:
> that tool would still be responsible for creating and deleting the temp file right?

Yeah, or it could read directly to memory from the remote source.  In my opinion either is valid, so I'd preference the way you have it here in the name of simplicity and expediency